schema/extension.go

```diff
@@ -57,13 +55,14 @@ type (
 
 func NewExtensionRunner(meta ExtensionRunnerMetaSchema, runners ...Extension) (*ExtensionRunner, error) {
```

Comment:
This is always `embedx.IdentityExtension`
```suggestion
func NewExtensionRunner(runners ...Extension) (*ExtensionRunner, error) {
```